test/spec/runners/epoch_processing.ex

```diff
@@ -68,18 +68,31 @@ defmodule EpochProcessingTestRunner do
     assert result == {:ok, post}
   end
 
-  defp handle_case("slashings_reset", pre, post) do
-    result = EpochProcessing.process_slashings_reset(pre)
-    assert result == {:ok, post}
-  end
-
   defp handle_case("randao_mixes_reset", pre, post) do
     result = EpochProcessing.process_randao_mixes_reset(pre)
     assert result == {:ok, post}
   end
 
+  defp handle_case("registry_updates", pre, post) do
+    result = EpochProcessing.process_registry_updates(pre)
+
+    case post do
+      nil ->
+        assert {output, _error_msg} = result
+        assert output == :error
```

Comment:
Matching this way we get better error messages:
```suggestion
        assert {:error, _error_msg} = result
```